sale_commission_geo_assign/README.rst

```diff
@@ -0,0 +1,98 @@
+===================================
+Sales commissions - Geo assignation
+===================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Production%2FStable-green.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Production/Stable
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fcommission-lightgray.png?logo=github
+    :target: https://github.com/OCA/commission/tree/10.0/sale_commission_geo_assign
+    :alt: OCA/commission
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/commission-10-0/commission-10-0-sale_commission_geo_assign
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/165/10.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+Configure sales agents assigning them to a specific geographical area.
+
+Then, automatically assign agents to customers, according to their geographical area.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+Go to
+
+Sales > Commissions Management > Agents
+
+For every agent, you can set Countries, States or ZIP range
```

Comment:
Can't find the fields in runbot:
![image](https://user-images.githubusercontent.com/32064796/47725564-10fcc080-dc59-11e8-851d-262ea8122abf.png)

Reply:
Indeed! I made my own tests locally as It's written in my review comment.

Reply:
This is because of https://github.com/OCA/commission/blob/5dd306358709a569509cde27954a11770d872b50/sale_commission_areamanager/views/res_partner_view.xml#L9 ...
I am going to fix

Reply:
@SimoRubi I made the other changes meanwhile

Reply:
@tafaRU @SimoRubi here it is https://github.com/OCA/commission/pull/184